dot/network/stream_manager_test.go

```diff
@@ -41,15 +37,22 @@ func setupStreamManagerTest(t *testing.T) (context.Context, []libp2phost.Host, [
 	require.NoError(t, err)
 
 	ha, err := libp2p.New(
-		ctx, libp2p.ListenAddrs(addrA),
+		libp2p.ListenAddrs(addrA),
 	)
 	require.NoError(t, err)
 
 	hb, err := libp2p.New(
-		ctx, libp2p.ListenAddrs(addrB),
+		libp2p.ListenAddrs(addrB),
 	)
 	require.NoError(t, err)
 
+	t.Cleanup(func() {
+		cleanupStreamInterval = time.Minute
```

Comment:
nit A bit out of scope, but having this horrendous global variable `cleanupStreamInterval` within the `streamManager` struct would be much nicer than a global var we fiddle with in tests